components/local-app/version.txt

```diff
@@ -0,0 +1 @@
+v0.1
```

Comment:
In some experiments I've done with this, I have added the `gitpodCLIVersion` property into the `WORKSPACE.yaml` file. Do you think it's better here?

Reply:
The benefits of making this a file in here are:
- We don't have to carry this through as build-args into all the individual builds.
- When the version changes, we're guaranteed to have a source code change, hence a rebuild.
- Because of Go Embed, using this version in a variety of places is pretty simple.
- It keeps the version close to where it's used, specifically in a local app package.

Reply:
Sounds good. I like the "keep close to where it's used" part